scripts/release/steps/validate-new-version.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const chalk = require("chalk");
+const semver = require("semver");
+
+module.exports = async function({ version, previousVersion }) {
+  if (!semver.valid(version)) {
+    throw Error(`Invalid version specified`);
+  }
+
+  if (!semver.gt(version, previousVersion)) {
+    throw Error(`Version ${chalk.white(version)} has already been published`);
```

Comment:
If the user has a light theme, the version number may not appear (unless the error message is shown with a colored background).

Reply:
Oh that was left when I was printing the error all in red.. gonna change this